Directory.Packages.props

```diff
@@ -171,6 +171,8 @@
     <PackageVersion Include="Azure.Identity" Version="1.13.2" />
     <!-- https://github.com/Azure/azure-cosmos-dotnet-v3/pull/3313 -->
     <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
+    <!-- VS telemetry package for dashboard telemetry -->
+    <PackageVersion Include="Microsoft.VisualStudio.Telemetry" Version="17.14.2" />
```

Comment:
What do we use from here? It has a fair number of dependencies.

Reply:
Just a few enums. I copied the enums and removed the dependency